From 1a4e1dece5c9814e2aa38a51e50ae9036f5a8775 Mon Sep 17 00:00:00 2001
From: Robert Marko <robert.marko@sartura.hr>
Date: Tue, 30 Jun 2026 11:23:47 +0200
Subject: [PATCH 71/73] arm64: dts: microchip: tactical-1000: add port names

Now that driver supports parsing the "label" property, populate port names
as they are physically wired up.

Signed-off-by: Robert Marko <robert.marko@sartura.hr>
(cherry picked from commit c4eb4408727549e4b0012646ab94e80a1992cd83)
---
 .../dts/microchip/lan9696-tactical-1000.dts   | 29 +++++++++++++++++++
 1 file changed, 29 insertions(+)

diff --git a/arch/arm64/boot/dts/microchip/lan9696-tactical-1000.dts b/arch/arm64/boot/dts/microchip/lan9696-tactical-1000.dts
index e9fa26e05804..e7e4654af98f 100644
--- a/arch/arm64/boot/dts/microchip/lan9696-tactical-1000.dts
+++ b/arch/arm64/boot/dts/microchip/lan9696-tactical-1000.dts
@@ -626,6 +626,7 @@ port0: port@0 {
 			phy-mode = "qsgmii";
 			phys = <&serdes 0>;
 			microchip,bandwidth = <1000>;
+			label = "lan4";
 		};
 
 		port1: port@1 {
@@ -634,6 +635,7 @@ port1: port@1 {
 			phy-mode = "qsgmii";
 			phys = <&serdes 0>;
 			microchip,bandwidth = <1000>;
+			label = "lan3";
 		};
 
 		port2: port@2 {
@@ -642,6 +644,7 @@ port2: port@2 {
 			phy-mode = "qsgmii";
 			phys = <&serdes 0>;
 			microchip,bandwidth = <1000>;
+			label = "lan2";
 		};
 
 		port3: port@3 {
@@ -650,6 +653,7 @@ port3: port@3 {
 			phy-mode = "qsgmii";
 			phys = <&serdes 0>;
 			microchip,bandwidth = <1000>;
+			label = "lan1";
 		};
 
 		port4: port@4 {
@@ -658,6 +662,7 @@ port4: port@4 {
 			phy-mode = "qsgmii";
 			phys = <&serdes 1>;
 			microchip,bandwidth = <1000>;
+			label = "lan8";
 		};
 
 		port5: port@5 {
@@ -666,6 +671,7 @@ port5: port@5 {
 			phy-mode = "qsgmii";
 			phys = <&serdes 1>;
 			microchip,bandwidth = <1000>;
+			label = "lan7";
 		};
 
 		port6: port@6 {
@@ -674,6 +680,7 @@ port6: port@6 {
 			phy-mode = "qsgmii";
 			phys = <&serdes 1>;
 			microchip,bandwidth = <1000>;
+			label = "lan6";
 		};
 
 		port7: port@7 {
@@ -682,6 +689,7 @@ port7: port@7 {
 			phy-mode = "qsgmii";
 			phys = <&serdes 1>;
 			microchip,bandwidth = <1000>;
+			label = "lan5";
 		};
 
 		port8: port@8 {
@@ -690,6 +698,7 @@ port8: port@8 {
 			phy-mode = "qsgmii";
 			phys = <&serdes 2>;
 			microchip,bandwidth = <1000>;
+			label = "lan12";
 		};
 
 		port9: port@9 {
@@ -698,6 +707,7 @@ port9: port@9 {
 			phy-mode = "qsgmii";
 			phys = <&serdes 2>;
 			microchip,bandwidth = <1000>;
+			label = "lan11";
 		};
 
 		port10: port@10 {
@@ -706,6 +716,7 @@ port10: port@10 {
 			phy-mode = "qsgmii";
 			phys = <&serdes 2>;
 			microchip,bandwidth = <1000>;
+			label = "lan10";
 		};
 
 		port11: port@11 {
@@ -714,6 +725,7 @@ port11: port@11 {
 			phy-mode = "qsgmii";
 			phys = <&serdes 2>;
 			microchip,bandwidth = <1000>;
+			label = "lan9";
 		};
 
 		port12: port@12 {
@@ -722,6 +734,7 @@ port12: port@12 {
 			phy-mode = "qsgmii";
 			phys = <&serdes 3>;
 			microchip,bandwidth = <1000>;
+			label = "lan16";
 		};
 
 		port13: port@13 {
@@ -730,6 +743,7 @@ port13: port@13 {
 			phy-mode = "qsgmii";
 			phys = <&serdes 3>;
 			microchip,bandwidth = <1000>;
+			label = "lan15";
 		};
 
 		port14: port@14 {
@@ -738,6 +752,7 @@ port14: port@14 {
 			phy-mode = "qsgmii";
 			phys = <&serdes 3>;
 			microchip,bandwidth = <1000>;
+			label = "lan14";
 		};
 
 		port15: port@15 {
@@ -746,6 +761,7 @@ port15: port@15 {
 			phy-mode = "qsgmii";
 			phys = <&serdes 3>;
 			microchip,bandwidth = <1000>;
+			label = "lan13";
 		};
 
 		port16: port@16 {
@@ -754,6 +770,7 @@ port16: port@16 {
 			phy-mode = "qsgmii";
 			phys = <&serdes 4>;
 			microchip,bandwidth = <1000>;
+			label = "lan20";
 		};
 
 		port17: port@17 {
@@ -762,6 +779,7 @@ port17: port@17 {
 			phy-mode = "qsgmii";
 			phys = <&serdes 4>;
 			microchip,bandwidth = <1000>;
+			label = "lan19";
 		};
 
 		port18: port@18 {
@@ -770,6 +788,7 @@ port18: port@18 {
 			phy-mode = "qsgmii";
 			phys = <&serdes 4>;
 			microchip,bandwidth = <1000>;
+			label = "lan18";
 		};
 
 		port19: port@19 {
@@ -778,6 +797,7 @@ port19: port@19 {
 			phy-mode = "qsgmii";
 			phys = <&serdes 4>;
 			microchip,bandwidth = <1000>;
+			label = "lan17";
 		};
 
 		port20: port@20 {
@@ -786,6 +806,7 @@ port20: port@20 {
 			phy-mode = "qsgmii";
 			phys = <&serdes 5>;
 			microchip,bandwidth = <1000>;
+			label = "lan24";
 		};
 
 		port21: port@21 {
@@ -794,6 +815,7 @@ port21: port@21 {
 			phy-mode = "qsgmii";
 			phys = <&serdes 5>;
 			microchip,bandwidth = <1000>;
+			label = "lan23";
 		};
 
 		port22: port@22 {
@@ -802,6 +824,7 @@ port22: port@22 {
 			phy-mode = "qsgmii";
 			phys = <&serdes 5>;
 			microchip,bandwidth = <1000>;
+			label = "lan22";
 		};
 
 		port23: port@23 {
@@ -810,6 +833,7 @@ port23: port@23 {
 			phy-mode = "qsgmii";
 			phys = <&serdes 5>;
 			microchip,bandwidth = <1000>;
+			label = "lan21";
 		};
 
 		port24: port@24 {
@@ -820,6 +844,7 @@ port24: port@24 {
 			managed = "in-band-status";
 			microchip,bandwidth = <10000>;
 			microchip,sd-sgpio = <24>;
+			label = "sfp1";
 		};
 
 		port25: port@25 {
@@ -830,6 +855,7 @@ port25: port@25 {
 			managed = "in-band-status";
 			microchip,bandwidth = <10000>;
 			microchip,sd-sgpio = <28>;
+			label = "sfp2";
 		};
 
 		port26: port@26 {
@@ -840,6 +866,7 @@ port26: port@26 {
 			managed = "in-band-status";
 			microchip,bandwidth = <10000>;
 			microchip,sd-sgpio = <32>;
+			label = "sfp3";
 		};
 
 		port27: port@27 {
@@ -850,6 +877,7 @@ port27: port@27 {
 			managed = "in-band-status";
 			microchip,bandwidth = <10000>;
 			microchip,sd-sgpio = <36>;
+			label = "sfp4";
 		};
 
 		port29: port@29 {
@@ -858,6 +886,7 @@ port29: port@29 {
 			phy-handle = <&phy3>;
 			phy-mode = "rgmii-id";
 			microchip,bandwidth = <1000>;
+			label = "mgmt";
 		};
 	};
 };
